### ISSUE TITLE
Windows directories are troublesome for repoman exec

### ISSUE THREAD

**Issue by Guysbert** [basti1302/repoman#20]
It appears that you can't use repoman exec with windows.

Simple example:

Assume you have the following structure:
```
D:\workspace\project-a
D:\workspace\project-b
```
The ```.repoman.json```can be found under D:\workspace. 

If you execute from there ```repoman exec 'dir'``` you get as result:
```
+ project-a
The system cannot find the path specified.

+ project-b
The system cannot find the path specified.
```

If you execute ```repoman exec 'cd {workspace}' -v``` you get as result:
```
+ project-a
> cd W:\workspace
The filename, directory name, or volume label syntax is incorrect.

+ project-b
> cd W:\workspace
The filename, directory name, or volume label syntax is incorrect.
```
I assume repoman as trouble with the windows backslash. Would be great if that could be fixed :)

**Comment by basti1302** [comment]
Hey Andreas,

thanks for the detailed report. Do you have a pull request for me? :innocent:

**Comment by Guysbert** [comment]
Oh you rascal. :stuck_out_tongue: 
Well, apparently you already commited a fix. I'd like to try that out soon. How do I use the latest... is there such a thing as snapshots here?

**Comment by basti1302** [comment]
No, the fix is not enough. The command for repoman exec is processed with some rather exotic string templating engine and it appers this engine does not play nicely with backslashes. I'll replace it with something else and try again. I'll let you know if I have something you can test.

**Comment by Guysbert** [comment]
Another note: Maybe it's not even necessary to replace the slash with a backslash. Usually I am using git bash to execute repoman under windows, where i knew it's possible to use slashes. But even the windows cmd knows how to interpret slashes.
Although there you have to explicitly change the drive first.
Examples:
[Screenshot git bash](https://www.dropbox.com/s/612fgo7bjvhtpu9/Screenshot%202016-01-30%2012.45.15.png?dl=0)
[Screenshot cmd](https://www.dropbox.com/s/yy6ucmqd08a4bna/Screenshot%202016-01-30%2012.41.41.png?dl=0)

**Comment by basti1302** [comment]
This should be fixed in 1.0.0.

For the record, the main problem was not the path separator (`/` versus `\`) but that repoman constructed something like `cd W:\workspace\repository-name; dir` and the `;` can not be used to separate multiple commands on one line in Windows, thus the whole of "W:\workspace\repository-name; dir" was interpreted as a directory name.

I did not do anything for the change-the-drive issue. I believe situations where .repoman.json is on a different drive than the individual repositories is rare, if this can occur at all. This would warrant a separate issue.

**Comment by Guysbert** [comment]
\o/
